tests/test_core.py

```diff
@@ -397,3 +397,14 @@ def test_geoseries_apply(geoseries_polygons):
 def test_geodataframe_html_repr(geodf_points):
     dask_obj = dask_geopandas.from_geopandas(geodf_points, npartitions=2)
     assert "Dask-GeoPandas GeoDataFrame" in dask_obj._repr_html_()
+
+
+def test_hilbert_distance(geodf_points):
```

Comment:
Let's maybe start a new test file for all tests related to calculating the hilbert distance

Reply:
ok

Reply:
resolved in 8737e55